source/_components/light.xiaomi_philipslight.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: "Xiaomi Philips Light"
+description: "Instructions how to integrate your Xiaomi Philips Lights within Home Assistant."
+date: 2017-08-26 08:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: philips.png
+ha_category: Light
+ha_version: 0.53
+ha_iot_class: "Local Polling"
+---
+
+The `xiaomi_philipslight` platform allows you to control the state of your Xiaomi Philips LED Ball Lamp and Xiaomi Philips LED Ceiling Lamp.
+
+Current supported features are `on`, `off`, `set_cct` (colortemp) , `set_bright` (brightness).
+
+## {% linkable_title Getting started %}
```

Comment:
I moved the instructions from the vacuum platform to the component. So a link to https://home-assistant.io/components/xiaomi/#retrieving-the-access-token would be sufficient.

Reply:
The new location of the token instructions isn't appropriate. The page "https://home-assistant.io/components/xiaomi/#retrieving-the-access-token" belongs to the xiaomi gateway. A device with another protocol and another security mechanism (key != token). We mixed up two independent platforms now.

Reply:
Same app, same name... Sure, we can improve the location but I don't see an issue here as the intro make it clear that it's for platforms.

Maintaining three times the same instructions which will be outdated probably soon and are extensive doesn't make sense and will lead to frustrated users. We saw that with the Raspberry platforms in the past.